src/TypesOfTreatments/Treatments.js

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import treatment1 from './assets/dryneedle.jpg';
+import treatment2 from './assets/physiotherapy.jpg';
+import treatment3 from './assets/posture.jpg';
+import treatment4 from './assets/footalign.jpg';
+import Row from 'react-bootstrap/Row';
+import "./treatment.css";
+
+function Treatments (){
+   return(
+        <>
+       <Row>
+         <Card className="mb-1" style={{ width: '40%' }}>
+      <Card.Img variant="top" src={treatment1} />
```

Comment:
Good. To use but we are just causing discrepancies in our sites. We should either use React Bootstrap or Bootstrap. Using both is not an issue just not a good coding practice.